src/urllib3/contrib/pyopenssl.py

```diff
@@ -450,7 +449,7 @@ def load_cert_chain(self, certfile, keyfile=None, password=None):
         self._ctx.use_privatekey_file(keyfile or certfile)
 
     def set_alpn_protocols(self, protocols):
-        protocols = [six.ensure_binary(p) for p in protocols]
+        protocols = [util.util.to_bytes(p) for p in protocols]
```

Comment:
nit: Let's expand the import to `from ..util.util import to_bytes` so it's similar to the rest of the usages

Reply:
doc tests will fail with this change.

Maybe it is related to `from .. import util` that exists at the top of file

Reply:
This should be `to_bytes(p, "ascii")`